page-data.php

```diff
@@ -100,8 +100,8 @@
         name: 'OIS - officers shot by civilians',
         description: "Shootings that injured or killed Texas law enforcement officers since Sept. 2015, as reported to the Office of the Attorney General.",
         urls: {
-          compressed: '<?php echo get_template_directory_uri(); ?>/data/ois_officers_compressed.json',
-          full: '<?php echo get_template_directory_uri(); ?>/data/ois_officers_full.csv',
+          compressed: 'https://s3.amazonaws.com/tji-compressed-data/ois_officers_compressed.json',
```

Comment:
Some of the charts and filters are missing here -- this means this data is probably missing in the supplied JSON. 
![screen shot 2019-02-11 at 12 00 26 am](https://user-images.githubusercontent.com/3039125/52551181-39d03e00-2d90-11e9-9da7-41fbea12ce01.png)
![screen shot 2019-02-11 at 12 00 33 am](https://user-images.githubusercontent.com/3039125/52551186-3d63c500-2d90-11e9-8322-65809e849d6a.png)

Reply:
Hmm, I wasn't aware these were added and when I made the PR those didn't show up on the page for me. @aidenyang Can you take a look at this and make the appropriate changes to the database?

Reply:
Hey @judan these charts have been on production for awhile - I think they were added 3 months ago — here’s the original PR for it.  https://github.com/texas-justice-initiative/website/pull/118 — you’ll see a note in that PR that says Everett needed to make changes to the data processing repo

Reply:
Hey @judan these charts have been on production for awhile - I think they were added 3 months ago — here’s the original PR for it.  https://github.com/texas-justice-initiative/website/pull/118 — you’ll see a note in that PR that says Everett needed to make changes to the data processing repo

Reply:
Hmm ok. Well, I'm not sure what is missing in the json file(s) I was given. Does Aiden need to update something in json, or do I need to fetch something else here so that the charts will appear correctly?

Reply:
@wetchler did you ever commit your OIS changes to the data processing repo you mentioned in this PR — https://github.com/texas-justice-initiative/website/pull/118